terraform/modules/hub/hub_config.tf

```diff
@@ -89,6 +89,7 @@ data "template_file" "config_task_def" {
     metadata_object_key      = local.metadata_object_key
     memory_hard_limit        = var.config_memory_hard_limit
     jvm_options              = var.jvm_options
+    service_log_level        = var.service_log_level
```

Comment:
Can we update the task definition JSON file to pick up this variable `service_log_level `?